### DISCUSSION TITLE
Servo acceleration and deceleration setting

### DISCUSSION THREAD

**Discussion by serzhantzevs4** [khoih-prog/ESP8266_ISR_Servo#2] [Ideas]
Hello. I express my deep gratitude to you for this development. I am very interested in maybe adding a very important function to this project. Namely, the speed of rotation of the servo. So in some projects the rattling of the servo can be avoided. I am sorry for my bad english. 
~ Setting the acceleration (acceleration and deceleration) of the servo
~ Soft start from any position when starting the program
https://github.com/GyverLibs/ServoSmooth

**Comment by khoih-prog** [comment]
Thanks a lots for your **very good** library :+1: 

It'll be great if you can rewrite the library a little bit or create a new library, in order to use any of my ISR_Servo libraries, such as 

1. [ESP8266_ISR_Servo](https://github.com/khoih-prog/ESP8266_ISR_Servo)
2. [ESP32_ISR_Servo](https://github.com/khoih-prog/ESP32_ISR_Servo)
3. [ESP32_S2_ISR_Servo](https://github.com/khoih-prog/ESP32_S2_ISR_Servo)
4. [ESP32_C3_ISR_Servo](https://github.com/khoih-prog/ESP32_C3_ISR_Servo)
5. [ESP32_New_ISR_Servo](https://github.com/khoih-prog/ESP32_New_ISR_Servo)
6. [STM32_ISR_Servo](https://github.com/khoih-prog/STM32_ISR_Servo)
7. [RP2040_ISR_Servo](https://github.com/khoih-prog/RP2040_ISR_Servo)
8. [SAMD_ISR_Servo](https://github.com/khoih-prog/SAMD_ISR_Servo)
9. [NRF52_ISR_Servo](https://github.com/khoih-prog/NRF52_ISR_Servo)
10. [Portenta_H7_ISR_Servo](https://github.com/khoih-prog/Portenta_H7_ISR_Servo)

The user then just includes both libraries from the sketch, such as

```
#include "ESP32_New_ISR_Servo.h"
#include <ServoSmooth.h>
```

then is able to use the good `smooth` feature of your library.

I'm waiting for your response.

Best Regards,
